src/pages/Autorised.js

```diff
@@ -0,0 +1,27 @@
+import { Route, Switch, Redirect } from 'react-router-dom';
+import { TaskList, Task, Main } from '../components';
```

Comment:
why are the pages in the components folder? If they are pages, they should be in pages folder